cmake/third_party.cmake

```diff
@@ -125,9 +148,16 @@ if(WITH_GPU)
 endif(WITH_GPU)
 
 if(WITH_PSLIB)
-    include(external/libmct)
-    include(external/pslib)
-    include(external/pslib_brpc)
+    include(external/pslib)          # download, build, install pslib
+    list(APPEND third_party_deps pslib)
+    if(WITH_LIBMCT)
+        include(external/libmct)     # download, build, install libmct
+        list(APPEND third_party_deps libmct)
```

Comment:
seems both "extern_xxx" and "xxx" are used in this file, can you unify the format?